### ISSUE TITLE
Patient Saturation Detection

### ISSUE THREAD

**Issue by nathanbunker** [immregistries/mqe#197]
Copied from AART-2033

Eric writes:

We need a new detection for Patient Saturation. The detection should be "Patient has at least 2 vaccination events before age 6 years". This would allow us to attempt to mimic IISAR "patient participation in an IIS" which has the 2+ dose qualifier.

**Comment by nathanbunker** [comment]
This is MQE0770. 

The count is of the number of events, not vaccinations. Two vaccinations given on the same day will only count as one event.

**Comment by larsonet9** [comment]
hmm... Sorry if I wasn't clear. Would it be possible to count shots? That's what I meant by Vaccination Event. Otherwise, we won't compare apples-to-apples with IISAR.

**Comment by nathanbunker** [comment]
Oh, okay. I'll have to make a patch and tell Hossam about this. Probably won't be a big deal to get this in.

**Comment by nathanbunker** [comment]
My understanding is that this would be helpful but not required for Assessment in 2022.

**Comment by larsonet9** [comment]
Correct. This won't be used by Assessment at all, but would be useful for the new Patient Saturation T&D.
